instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-payara-testing/src/test/groovy/PayaraRestTest.groovy

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+class PayaraRestTest extends ArquillianRestTest {
```

Comment:
Should `ArquillianRestTest` be abstract? Since it's not supposed to run by itself, it's to be extended.